flipt-client-browser/src/models.ts

```diff
@@ -0,0 +1,63 @@
+export type Authentication = string;
+
+export function WithClientTokenAuthentication(client_token: string) {
+  return `Bearer ${client_token}`;
+}
+
+export function WithJWTAuthentication(jwt_token: string) {
+  return `JWT ${jwt_token}`;
+}
```

Comment:
Is it more common to lower-case these as they're functions?
Im unsure what makes most sense to folks, just looked quite like our Go one and I wondered if that is expected in TS land.

Reply:
yeah, tbh i dont love this. I might try and make it like our existing `flipt-client-node` auth where its simply an object config like:

```
"authentication": {
  "client_token": "secret"
}
```